### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
updating for consistency with template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
 The javascript.client project implements the [WebSocket standard](https://tools.ietf.org/html/rfc6455) in JavaScript. It provides a WebSocket client API and an AMQP client API that enables developers to build JavaScript applications that communicate with an RFC 6455 endpoint or an AMQP message broker over WebSocket.
 
-# Building this Project
+# Building This Project
 
-##Installing Dependencies
-Before building kaazing-client-javascript, please install and configure the following dependencies:
+## Installing Dependencies
+Before building the kaazing-client-javascript, please install and configure the following dependencies:
```
Comment (brennangaunce) [inline README.md]: Before building the kaazing-client-javascript _repo_?

File: README.md [hunk 2]
```diff
@@ -30,13 +29,13 @@ Before building kaazing-client-javascript, please install and configure the foll
 
 You can also download and run this project from [kaazing.org/download](http://kaazing.org/download/)
 
-# Learning How to Develop Client Applications
+# Learning How to Use an RFC 6455 endpoint
 
-To learn how to develop client applications using the Gateway, see the documentation on [kaazing.org](http://kaazing.org).  To contribute to the documentation source, see the doc directory under each protocol's folder.
+You can use an RFC-6455 endpoint, such KAAZING Gateway. To learn how to administer the Gateway, its configuration files, and security, see the documentation on [developer.kaazing.com](http://developer.kaazing.com/documentation/5.0/index.html).
```
Comment (brennangaunce) [inline README.md]: We should use 'latest' for this instead of a specific version.

Comment (ghost) [inline README.md]: The problem is that if you look at the top-level doc page then they have to figure out which edition to read: http://developer.kaazing.com/documentation/. Better to link them to the correct doc set, imo.

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md]: removed "the". I think the project name is sufficient.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # javascript.client
 
-# About this Project
+# About This Project
 
 The javascript.client project implements the [WebSocket standard](https://tools.ietf.org/html/rfc6455) in JavaScript. It provides a WebSocket client API and an AMQP client API that enables developers to build JavaScript applications that communicate with an RFC 6455 endpoint or an AMQP message broker over WebSocket.
```
Comment (vjwang) [inline README.md]: Add AMQP 0-9-1

File: README.md (hunk 2 above)
Comment (vjwang) [inline README.md]: This section can be used as the template across all CE client repos (to replace what you have currently, which is specific to the Gateway). See ios.client, c.client, and java.client.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
```
Comment (vjwang) [inline README.md]: Add AMQP 0-9-1

**Pull request merged**